src/authentication.ts

```diff
@@ -21,12 +21,14 @@ class CustomisedAuthenticationService extends AuthenticationService {
   async getPayload(authResult: AuthenticationResult, params: AuthenticationParams) {
     const payload = await super.getPayload(authResult, params)
     const { user } = authResult as { user: User }
+
     if (user) {
       payload.userId = user.uid
       if (user.groups.length) {
-        payload.userGroups = user.groups.map(d => d.name)
+        payload.groups = user.groups.map(d => d.name)
```

Comment:
[Here](https://github.com/impresso/impresso-middle-layer/pull/419/files#diff-d40228c94ec34f9469d59441e60c0b9b54b6a5be0953f01752e758b9489e16a0R98) it's still read as `userGroups`. Better leave the property as is - it's not typed.